test/testenv/util.go

```diff
@@ -349,3 +349,75 @@ func dumpGetPods(ns string) {
 		logf.Log.Info(line)
 	}
 }
+
+// CheckMCPodReady check if monitoring pod is ready
+func CheckMCPodReady(ns string) bool {
+	output, err := exec.Command("kubectl", "get", "pod", "-n", ns).Output()
+	if err != nil {
+		logf.Log.Error(err, "Failed to execute command", "command", "kubectl get pods")
+	}
+	for _, line := range strings.Split(string(output), "\n") {
+		if strings.Contains(line, "monitoring-console") {
+			slices := strings.Fields(line)
+			logf.Log.Info("MC Pod Found", "POD", slices[0], "STATUS", slices[1])
+			return strings.Contains(slices[1], "1/1")
+		}
+	}
+	return false
+}
+
+// GetMCPodCount get count of Monitoring Console in a namespace
+func GetMCPodCount(ns string) int {
+	output, err := exec.Command("kubectl", "get", "pod", "-n", ns).Output()
+	if err != nil {
+		logf.Log.Error(err, "Failed to execute command", "command", "kubectl get pods")
+	}
+	count := 0
+	for _, line := range strings.Split(string(output), "\n") {
+		if strings.Contains(line, "monitoring-console") {
+			count = count + 1
+		}
+	}
+	return count
+}
+
+// GetMCPodName returns name of monitoring console pod
+func GetMCPodName(ns string) string {
+	output, err := exec.Command("kubectl", "get", "pod", "-n", ns).Output()
+	if err != nil {
+		logf.Log.Error(err, "Failed to execute command", "command", "kubectl get pods")
+	}
+	podName := ""
+	for _, line := range strings.Split(string(output), "\n") {
+		if strings.Contains(line, "monitoring-console") {
+			slices := strings.Fields(line)
+			logf.Log.Info("MC Pod Found", "POD", slices[0], "STATUS", slices[1])
+			return slices[0]
+		}
+	}
+	return podName
+}
+
+// GetConfiguredPeers get list of Peers Confiugred on Montioring Console
+func GetConfiguredPeers(ns string) []string {
+	podName := GetMCPodName(ns)
+	var peerList []string
+	if len(podName) > 0 {
+		peerFile := "/opt/splunk/etc/apps/splunk_monitoring_console/local/splunk_monitoring_console_assets.conf"
+		output, err := exec.Command("kubectl", "exec", "-n", ns, podName, "--", "cat", peerFile).Output()
+		if err != nil {
+			logf.Log.Error(err, "Failed to execute command", "command", "kubectl exec -- cat /opt/splunk/etc/apps/splunk_monitoring_console/local/splunk_monitoring_console_assets.conf")
```

Comment:
Shouldn't the command string in the error be "kubectl exec -n ns podName -- cat /opt/splunk/etc/apps/splunk_monitoring_console/local/splunk_monitoring_console_assets.conf"